maps/map_files/Kutjevo/Kutjevo.dmm

```diff
@@ -8838,6 +8838,12 @@
 /obj/structure/largecrate/random/case/double,
 /turf/open/floor/kutjevo/colors/orange,
 /area/kutjevo/exterior/Northwest_Colony)
+"mfs" = (
```

Comment:
- Same as before, use Noop turf/area when there are no changes between a small area on the nightmare insert and base map.
- Damaged Barricades and Health.
- Ammunition and Guns, too much plus too collected to reflect a last stand area. Spread them out among corpses etc.
- Way too many spawn landmarks, as before spread them out a bit more and less.